docs/management/managing-indices.asciidoc

```diff
@@ -107,20 +132,35 @@ template before you create the indices.
 
 In this example, you’ll create an index template for randomly generated log files.
```

Comment:
Instead of the title "Example: Create an index template", we've been using "Try it: Create an index template".  I like that better because it implies an action.  However, it hasn't been officially adopted.  Also, should this section be rewritten as steps?